en_US/rule/rule-engine.md

```diff
@@ -153,7 +153,7 @@ The SQL statements of the rule engine can handle both messages (message publishi
 The topic of the event message starts with `"$events/"`, such as `"$events/client_connected",` `"$events/session_subscribed"`.
 If you want emqx to publish the event message, you can configure it in the `emqx_rule_engine.conf` file.
 
-For all supported events and available fields, please see  [rule event](#rule-sql-events).
+For all supported events and available fields, please see [Event topics available for FROM clause](#event-topic-available-for-from-clause).
```

Comment:
This link may be broken (in the preview, it does not go to the corresponding section)

Reply:
I changed the link because it was broken before so it will at least not be worse then before. I think we can commit this and see if it works after the doc has been built.